src/list.js

```diff
@@ -17,6 +20,11 @@ export default class List extends React.Component {
   }
 
   render() {
+    console.log(this.props);
+    
+    if(this.props.isDelete === true){
```

Comment:
why you calling a function inside render, not good practice , you can move it to componetWillReciveProps,